models/models.reviews.js

```diff
@@ -1,14 +1,73 @@
 const db = require("../db/connection");
 
-exports.selectReviews = () => {
-	const query = `
+const validSortBys = [
+	"owner",
+	"title",
+	"review_id",
+	"category",
+	"review_img_url",
+	"created_at",
+	"votes",
+	"designed",
+	"comment_count"
+];
+const validOrders = ["asc", "desc"];

```

Comment:
This needs to be moved out of the global and into the function, as you don't want to use it anywhere else.